wca-regulations.md

```diff
@@ -192,6 +192,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 5b5c) If more than one part with one colored face is affected, the puzzle is considered unsolved (DNF).
         - 5b5d) If one or more parts with more than one colored face are affected, the puzzle is considered unsolved (DNF).
         - 5b5e) Regulations [5b5c](regulations:regulation:5b5c) and [5b5d](regulations:regulation:5b5d) supersede [5b5a](regulations:regulation:5b5a) and [5b5b](regulations:regulation:5b5b).
+    - 5b6) Definition: A part is a subset of puzzle components that can physically hold together (e.g. an individual piece, cap, single sticker, or sticker attached to another part).
```

Comment:
```suggestion
    - 5b6) Definition: A part is an individual puzzle component (e.g. a single sticker, a 3x3x3 center cap) or a group of puzzle components that remain connected and do not move relative to each other during ordinary operation of the puzzle (e.g. a 3x3 corner piece, a pyraminx tip).
```

I think this is a little clearer in terms of what the change appears trying to accomplish. I am also concerned about how considering both smaller components and larger parts as 'parts' could impact the application of 5b5, where a defect could be analysed differently depending on the level considered.

Reply:
I think this is pretty clear - there's still a weird edge case of having 2/3 of a corner piece, which is still considered a part though. Do you have any alternative wording for that?

Reply:
```suggestion
    - 5b6) Definition: A part is one or more fully connected components from a group of puzzle components that remain connected and do not move relative to each other during ordinary operation of the puzzle (e.g. a 3x3x3 centre cap, a pyraminx tip, a 3x3x3 corner piece missing one face, one face of a 3x3x3 corner piece).
```

This is getting super-awkward, but I've given it a go anyway.

Reply:
Maybe it's worth putting it into a separate guideline? I feel like there are too much examples for the regulation, but at the same time they all are important to understand the meaning.

Reply:
i think having a lot of examples is fine for this

Reply:
Yeah, I mean the examples are fine, but here's a bunch of them. To not make regs longer and longer, maybe just create a guideline with the label `[EXAMPLE]`?